app/src/main/res/layout/context_menu_title.xml

```diff
@@ -1,15 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <TextView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:maxLines="3"
-    android:gravity="center_vertical"
     android:ellipsize="end"
+    android:fontFamily="@string/font_roboto_regular"
+    android:gravity="center_vertical"
+    android:maxLines="3"
+    android:padding="24dp"
+    android:textColor="#ffb2b2b2"
```

Comment:
Where did you get this color from? I see an incorrect value in the mocks and I'm not sure this is the same color as the url text. I asked antlam for clarification in https://github.com/mozilla-mobile/focus-android/issues/1260#issuecomment-336595240 and I pinged him to respond soon!

Reply:
That was specified in the mock.

![image](https://user-images.githubusercontent.com/19215803/31572218-30eaac90-b090-11e7-9e3d-f1de564755f6.png)